src/lib/formatter/RuleResultRecombinator.ts

```diff
@@ -274,22 +282,31 @@ URL: ${url}
 		return Mustache.render(template, templateData);
 	}
 
-	private static async constructCsv(results: RuleResult[]): Promise<string> {
+	private static async constructCsv(results: RuleResult[], normalizeSeverity: boolean): Promise<string> {
 		// If the results were just an empty string, we can return it.
 		if (results.length === 0) {
 			return '';
 		}
 
 		const csvRows = [];
-		// Gradually build our CSV, starting with these columns.
-		csvRows.push(['Problem', 'File', 'Severity', 'Line', 'Column', 'Rule', 'Description', 'URL', 'Category', 'Engine']);
-		let problemCount = 0;
 
+		if (normalizeSeverity){
```

Comment:
In order to avoid maintaining the two separate arrays, this could be 

```Javascript
        const columns: string[] = ['Problem', 'File', 'Severity']; 
        if (normalizeSeverity) {
           columns.push('Normalized Severity')
        }
        columns.push('Line', 'Column', 'Rule', 'Description', 'URL', 'Category', 'Engine');
        csvRows.push(columns);
```

The same approach could applied for each row.